components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/VariantListController.java

```diff
@@ -78,7 +78,7 @@
 
     private static final String INTERNAL_VARIANT_KEY = "cdna";
 
-    private static final String INTERNAL_GENESYMBOL_KEY = "genesymbol";
+    private static final String INTERNAL_GENESYMBOL_KEY = "gene";
```

Comment:
These are no longer gene symbols, so the name of this constant should be changed.